src/ElementBlock/src/Block.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { css, StyleSheet } from 'aphrodite';
+import { getStyles } from './getStyles';
+import { defaultConfig } from './configs';
+import { ElementBlockProps, ObjectLiteral } from './types';
+
+const Block = (props: ElementBlockProps) => {
+    const { globalSettings, customAttrs, children } = props;
+    const styles = StyleSheet.create(getStyles(props, globalSettings));
+    const customProps: ObjectLiteral = {};
+    customAttrs.forEach((attr: { name: string; value: string }) => {
+        const hyphenateString = (string = '') => {
+            // const specialChars = /[^a-zA-Z0-9\s-_]+/g;
+            const nonAlphaNumericChars = /[^a-zA-Z0-9]+/g;
+            const attr = string
+                .trim()
+                // .replace(specialChars, '')
+                .replace(nonAlphaNumericChars, '-');
+            return attr;
+        };
+        const fullAttrName = `data-element-${hyphenateString(attr.name)}`;
+        customProps[fullAttrName] = attr.value;
```

Comment:
Using `map` & `reduce` here will get rid of the eslint security warning triggered by this line, something like this:
```tsx
    const attrs = customAttrs.map((attr: { name: string; value: string }) => {
        const hyphenateString = (string = '') => {
            const nonAlphaNumericChars = /[^a-zA-Z0-9]+/g;
            const attr = string
                .trim()
                // .replace(specialChars, '')
                .replace(nonAlphaNumericChars, '-');
            return attr;
        };
        return {
            [`data-element-${hyphenateString(attr.name)}`]: value
        }
    })
    .reduce((acc, attr) => {
        return { ...acc, ...attr }
    }, {})
```

Reply:
This issue seems to have disappeared...